client/swagger.yaml

```diff
@@ -353,19 +353,14 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
-  /vsocks/{id}:
+  /vsock:
     put:
-      summary: Creates new vsock with ID specified by the id parameter.
+      summary: Creates new vsock with ID specified in request body.
       description:
         If the vsock device with the specified ID already exists, its body will
         be updated based on the new input. May fail if update is not possible.
```

Comment:
I believe the swagger.yaml works now, but I did notice it's not an exact copy of the swagger.yaml from the firecracker repo (the description is different for `/vsock`): https://github.com/firecracker-microvm/firecracker/blob/v0.19.0/api_server/swagger/firecracker.yaml#L359-L362

Even though the differences are now just superficial, can we for simplicity just make this file be an exactly copy of the 0.19 branch's yaml file? It just makes it much easier to verify there are no differences. 

(I think in the long-run we should consider just directly consuming the yaml from the firecracker repo as a submodule in order to make this simpler, but that's way out of scope for this PR)

Reply:
Sure, done.